core/cli/src/main/java/mil/nga/giat/geowave/core/cli/operations/config/options/ConfigOptions.java

```diff
@@ -50,19 +49,19 @@
 		"-cf",
 		"--config-file"
 	}, description = "Override configuration file (default is <home>/.geowave/config.properties)")
-	private String configFile;
+	private static String configFile;
```

Comment:
we should move away from using a static reference here...in practice in may not be an issue, but for example if there ever were different instances of ConfigOptions that used different values for configFile, this static member variable would be a major problem.